lib/utils.nix

```diff
@@ -130,6 +130,9 @@ rec {
 
   ifNonNull' = x: y: if (x == null) then null else y;
 
+  isTrue = x: lib.isBool x && x;
+  isFalse = x: lib.isBool x && !x;
```

Comment:
If we do go down this route, could we have some doc comments and some motivational examples?

```md
    A predicate that tests if a value is exactly true, i.e. `v == true`.

    Useful when working with nullable booleans.

    #Examples

    ```
    isTrue null
    => false
    ```

    ```
    isTrue true
    => true
    ```

    ```
    isTrue "true"
    => false
    ```

    # Type

    ```
    Any -> Boolean
    ```
```

Reply:
No real opinion on it

Reply:
I prefer using simpler constructs like `==` than ad-hoc functions where possible